src/Core/Utils/Color.hpp

```diff
@@ -37,6 +38,32 @@ class ColorBase : public Eigen::Matrix<_Scalar, 4, 1> {
     /// cast operator, mandatory to use Vector arithmetic
     operator VectorType() { return *this; }
 
+    /// convert the color expressed in sRGB color space to linear RGB
+    static inline ColorBase toLinRGB(const ColorBase &srgb) {
```

Comment:
```suggestion
    static inline ColorBase sRGBToLinear(const ColorBase &srgb) {
```

Reply:
the proposal of @dlyr for the naming is better. I'll integrate ASAP